tests/testfiles/CommonMark_0.30/backslash_escapes/008.test

```diff
@@ -5,7 +5,6 @@
 %   
 %   ---\RESULT---
 
-\def\markdownOptionFencedCode{true}
```

Comment:
Options `breakableBlockquotes`, `fencedCode`, and `html` should be enabled by default for CommonMark compliance.